src/payment/strategies/index.ts

```diff
@@ -9,6 +9,7 @@ export { default as SagePayPaymentStrategy } from './sage-pay-payment-strategy';
 export { AfterpayPaymentStrategy } from './afterpay';
 export { AmazonPayPaymentStrategy, AmazonPayPaymentInitializeOptions } from './amazon-pay';
 export { BraintreeCreditCardPaymentStrategy, BraintreePaymentInitializeOptions, BraintreePaypalPaymentStrategy, BraintreeVisaCheckoutPaymentStrategy, BraintreeVisaCheckoutPaymentInitializeOptions } from './braintree';
```

Comment:
One line per class please

Reply:
This isn't a change in PR, but I'm gonna change it